docs/high/dataset.rst

```diff
@@ -261,6 +261,34 @@ safe to use with very large target selections.  It is supported for the above
    more information.
 
 
+.. warning::
+    Dataset item getter returns new numpy.ndarray. Some might find the following behavior surprising:
+
+   >>> a = 1
+   >>> b = np.zeros(shape=(2,2))
```

Comment:
b is unused? And I think it's clearer for the examples to use a literal number `1` rather than a variable `a`.